flixel/system/FlxSound.hx

```diff
@@ -94,12 +94,10 @@ class FlxSound extends FlxBasic
 	 */
 	public var volume(get, set):Float;
 
-	#if (sys && openfl_legacy)
```

Comment:
We may to add a new FlxDefine for FLX_PITCH, defined as `#if ((sys && openfl_legacy) || (lime > 3.10.0 && !flash))` or whatever version of lime is released next